src/main/java/com/sevenmap/App.java

```diff
@@ -42,6 +45,9 @@
 
     private Item testElement = new Item(new Vector3f(0, 0, -1.0f), new Vector3f(0, 0, 0), new Vector3f(1, 1, 1), mesh);
 
+    public static Logger logger = null;
```

Comment:
Why do you initialize your logger this way ? My guess is you should call the default constructor (or simply create one)

Reply:
can you please modify this ?